scriptmodules/emulators/pcsx2.sh

```diff
@@ -11,42 +11,54 @@
 
 rp_module_id="pcsx2"
 rp_module_desc="PS2 emulator PCSX2"
-rp_module_help="ROM Extensions: .bin .iso .img .mdf .z .z2 .bz2 .cso .ima .gz\n\nCopy your PS2 roms to $romdir/ps2"
+rp_module_help="ROM Extensions: .bin .iso .img .mdf .z .z2 .bz2 .cso .ima .gz\n\nCopy your PS2 roms to $romdir/ps2\n\nCopy the required BIOS file to $biosdir"
 rp_module_licence="GPL3 https://raw.githubusercontent.com/PCSX2/pcsx2/master/COPYING.GPLv3"
 rp_module_section="exp"
 rp_module_flags="!arm"
 
 function depends_pcsx2() {
     if isPlatform "64bit"; then
-        iniConfig " = " '"' "$configdir/all/retropie.cfg"
-        iniGet "own_sdl2"
-        if [[ "$ini_value" != "0" ]]; then
-            if dialog --yesno "PCSX2 cannot be installed on a 64bit system with the RetroPie custom version of SDL2 installed due to version conflicts with the multiarch i386 version of SDL2.\n\nDo you want to downgrade to your OS version of SDL2 and continue to install PCSX2?" 22 76 2>&1 >/dev/tty; then
-                chown $user:$user "$configdir/all/retropie.cfg"
-                if rp_callModule sdl2 revert; then
-                    iniSet "own_sdl2" "0"
-                else
-                    md_ret_errors+=("Failed to install $md_desc")
-                fi
-            else
-                md_ret_errors+=("$md_desc install aborted.")
-            fi
-        fi
+        # We need to add the target architecture (no side effects if it's already added)
+        dpkg --add-architecture i386
+        # Installing compiler dependencies for crossbuild
+        getDepends gcc-multilib g++-multilib
     fi
+    # Build dependencies (from the Debian/Ubuntu package: https://github.com/PCSX2/pcsx2/blob/master/debian-packager/control)
+    local depends=(cmake libaio-dev:i386 libasound2-dev:i386 libbz2-dev:i386 libgl1-mesa-dev:i386 libglu1-mesa-dev:i386 libgtk2.0-dev:i386 liblzma-dev:i386 libpng-dev:i386 libpulse-dev:i386 libpcap0.8-dev:i386 libsdl2-dev:i386 libsoundtouch-dev:i386 libwxbase3.0-dev:i386 libwxgtk3.0-dev:i386 libx11-dev:i386 libxml2-dev:i386 portaudio19-dev:i386 zlib1g-dev:i386 libasound2-plugins:i386 libusb-0.1-4:i386)
+    getDepends "${depends[@]}"
```

Comment:
Almost - the local depends needs to be defined at the top of the function, then `getDepends gcc-multilib g++-multilib` needs to be depends=(gcc-multilib g++-multilib)

and the next one should be `depends+=(cmake libaio-dev:i386 libasound2-dev)` etc

Reply:
actually, that's probably best

```
local depends=(cmake libaio-dev:i386 libasound2-dev:i386 libbz2-dev:i386 libgl1-mesa-dev:i386 libglu1-mesa-dev:i386 libgtk2.0-dev:i386 liblzma-dev:i386 libpng-dev:i386 libpulse-dev:i386 libpcap0.8-dev:i386 libsdl2-dev:i386 libsoundtouch-dev:i386 libwxbase3.0-dev:i386 libwxgtk3.0-dev:i386 libx11-dev:i386 libxml2-dev:i386 portaudio19-dev:i386 zlib1g-dev:i386 libasound2-plugins:i386 libusb-0.1-4:i386)
...
depends+=(gcc-multilib g++-multilib)
...
getDepends "${depends[@]}"
}
```

Reply:
Really sorry for that. Done.